apps/video/test/marionette/lib/video.js

```diff
@@ -130,14 +134,79 @@ Video.prototype = {
    * @return {Marionette.Element} First element of all thumbnail images.
    */
   get thumbnail() {
-    return this.client.findElement(Video.Selector.thumbnail);
+    return this.client.helper.waitForElement(Video.Selector.thumbnail);
   },
 
   /**
    * @return {Marionette.Element} video player element.
    */
   get player() {
-    return this.client.findElement(Video.Selector.player);
+    return this.client.helper.waitForElement(Video.Selector.player);
+  },
+
+  /**
+   * When a video begins playing, displays the controls.
+   * Function contains logic specific to when a video begins playing;
+   * not to be used to arbitrarily display the controls during the playing
+   * of a video.
+   */
+  waitForControls: function() {
+    var elapsedElem = this.client.findElement(Video.Selector.elapsedTime);
+
+    // Ensure when we tap on the player view the video controls are not
+    // displayed -- they are displayed when the video first plays and then
+    // they are automatically hidden.
+    //
+    // Wait for controls to appear as the video begins to play. Specify
+    // a timeout to account for the controls having already been displayed
+    // and auto-hidden.
+    this.client.waitFor(function()
+      { return elapsedElem.displayed(); },
```

Comment:
nit: This is awkward formatting because the function body looks just like the object literal below it.  If you can fit this on the line above, I think that would be best.  Or at least put the { on the line above